src/content/docs/en/guides/assets.mdx

```diff
@@ -50,7 +62,26 @@ import logo from '../assets/images/logo.png';
 <Image src={logo} alt="Astro" />
 ```
 
-### Content Collections integration
+## New Markdown support
+
+Images stored in `src/assets/` can be referenced in Markdown and MDX files. These will automatically be optimized and hashed by Astro's build process.
+
+Remote images are also valid, but are not automatically optimized by Astro's build process.
+
+```md
+<!-- src/pages/post-1.md -->
+# My Markdown Page
+
+<!-- Local image stored at public/assets/stars.png -->
+![A starry night sky.](../../assets/stars.png)
```

Comment:
You can't refer to images in `public` this way, this follow the normal rules so you'd have to do `/assets/stars.png` and it would count as a remote image (not sure if it was even intended that the image is in public?)

Reply:
Whoops, yeah that was an accident